node/src/Transport.ts

```diff
@@ -81,9 +81,23 @@ export interface TransportTraceEventData
 
 export type SctpState = 'new' | 'connecting' | 'connected' | 'failed' | 'closed';
 
+export type TransportBaseEvents = { routerclose: []; trace: [TransportTraceEventData] }
```

Comment:
No need for "base" in the name:

```suggestion
export type TransportEvents = { routerclose: []; trace: [TransportTraceEventData] }
```